src/components/UserInfo.js

```diff
@@ -11,7 +12,12 @@ export default class UserInfo {
   }
 
   setUserInfo(data) {
-    this._name.textContent = data.title;
-    this._description.textContent = data.subtitle;
+    this._name.textContent = data.name;
+    this._description.textContent = data.about;
+    this.setAvatar(data);
```

Comment:
Please remove this line. The avatar should be handled separately, since it has a different form. When initializing the user info you can call two methods.